examples/expense.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Server rent",
+  "description": "Dedicated server: andromeda.kosmos.org, April 2020",
```

Comment:
I was pondering making this an array, so that it's easier to have more structured yet flexible descriptions. Not sure.

Reply:
I think a description is good. but maybe we can add a metadata hash, that allows us to optionally add data like invoice IDs, contract details, etc.?

Reply:
If you want to render that stuff, you'd have to then introduce non-standardized support for custom objects. I'd prefer to have as much as possible standardized by the schema.

The idea with the array of strings here was that you could add a slight structure to descriptive details by rendering them with line breaks and such for example.

Reply:
... however, in order to add random details (similar to what we do with contributions at the moment), we could just set the global "additionalProperties" to `true` instead of `false`.

Reply:
I don't see an advantage in having an array of strings over a simple string as description. 
Either it's a description or some more structured data (key: value). 

additional information could be solely for documentation in the IPFS document. like we do it with the pullrequest/issue data.

Reply:
The advantage is that you don't need any extra custom implementation to merely render some structured details. In this example, one could think of "Dedicated server: andromeda.kosmos.org" plus "Monthly rent for: April 2020" plus whatever else would be interesting, but require more design than worth it for actual structured properties.

But I'll just leave it as a string then, and allow `additionalProperties`.